test/random.go

```diff
@@ -42,7 +42,7 @@ func SecureRandomAlphaString(length int) string {
 // SecureRandomBytes returns the requested number of bytes using crypto/rand
 func SecureRandomBytes(length int) []byte {
 	var randomBytes = make([]byte, length)
-	_, err := rand.Read(randomBytes)
+	_, err := r.Read(randomBytes)
```

Comment:
crypto.rand should likely be kept. math.rand is less random

Reply:
Weird, docs say the opposite: 

```This package's outputs might be easily predictable regardless of how it's seeded. For random numbers suitable for security-sensitive work, see the crypto/rand package. 
```
seems like this is only for tests though, so either is prob fine.

Reply:
Ah, yeah the linter is saying that math.rand is weaker. It just will only complain if both are imported.